pallets/logion_loc/src/lib.rs

```diff
@@ -284,6 +290,50 @@ pub mod pallet {
 				}
 			}
 		}
+
+		/// Make a LOC void.
+		#[pallet::weight(T::WeightInfo::make_void())]
+		pub fn make_void(
+			origin: OriginFor<T>,
+			#[pallet::compact] loc_id: T::LocId,
+			reason: Vec<u8>,
+			replacer_loc_id: Option<T::LocId>
+		) -> DispatchResultWithPostInfo {
+			let who = ensure_signed(origin)?;
+
+			if !<LocMap<T>>::contains_key(&loc_id) {
+				Err(Error::<T>::NotFound)?
+			} else {
+				let loc = <LocMap<T>>::get(&loc_id).unwrap();
+				if loc.owner != who {
+					Err(Error::<T>::Unauthorized)?
+				}
+			}
+
+			if replacer_loc_id.is_some() {
+				let replacer = replacer_loc_id.unwrap();
+				if !<LocMap<T>>::contains_key(&replacer) {
+					Err(Error::<T>::ReplacerLocNotFound)?
+				} else {
+					let replacer_loc = <LocMap<T>>::get(&replacer).unwrap();
+					if replacer_loc.owner != who {
+						Err(Error::<T>::Unauthorized)?
+					}
+				}
+			}
+			let replacer = replacer_loc_id.map(|id| LocLink { id, nature: "replacer".as_bytes().to_vec() });
```

Comment:
As discussed, I am not really a fan of hard coding an artificial nature here. Let's ask Elie what he thinks about this (does it make sense to add a nature to the link?).

Reply:
> As discussed, I am not really a fan of hard coding an artificial nature here. Let's ask Elie what he thinks about this (does it make sense to add a nature to the link?).

The LocLink and thus the nature was removed in [this PR](https://github.com/logion-network/logion-node/pull/20).